cli/crates/typed-resolvers/Cargo.toml

```diff
@@ -9,7 +9,15 @@ repository.workspace = true
 publish = false
 
 [dependencies]
-graphql-parser = "0.4"
+async-graphql-parser = "6"
+async-graphql-value = "6"
+walkdir = "2"
+
+swc_ecma_parser = "0.141.2"
+swc_ecma_ast = "0.110.0"
+swc_common = "0.33"
+miette = { version = "5.10.0", features = ["fancy"] }
```

Comment:
:3

Reply:
I learned this library for this PR, and I think I'm going to keep using it, the APIs make sense and the errors are very nice.

Reply:
we should consider it for graphql parsing too...

Reply:
Absolutely